### DISCUSSION TITLE
Lolin S2 mini no AP

### DISCUSSION THREAD

**Discussion by lassekite** [micropython/micropython#9960] [Hardware & Peripherals]
Just bought two Lolin S2 mini and none will show up as AP, as normaly done I start up with 
`import webrepl_setup`
One is with the original fw 1.18 and one upgraded with 1.19, same behavier.
I can get an AP if I setup it my self like
`import network`
`ap = network.WLAN(network.AP_IF)`
`ap.config(ssid='MircoPython-S2-A')`
`ap.active(True)`

Should not the S2 behave like D1 or have I missed something?
Also if I connect it to my WIFI net it got some SSID like "espessif"

**Comment by davefes** [comment]
Is the S2 an ESP32, if so then it behaves differently than the ESP8266

**Reply by lassekite** [reply]
It's an ESP8266

**Reply by davefes** [reply]
Maybe they are different chip revisions.  I recall some improvements were made to the ESP32 wifi recently, maybe that resulted in some changes to the ESP8266 firmware.

Sorry, I am focused on the ESP32 and was aware that WiFi startup was different on the two devices.

**Reply by lassekite** [reply]
FYI I have tested with version 1.18,that was the original fw and 1.19. Same thing

**Reply by fivdi** [reply]
> It's an ESP8266

It's an ESP32-S2. See https://github.com/wemos/docs/blob/master/docs/en/s2/s2_mini.rst.

**Comment by fivdi** [comment]
In addition to what you have already done with `import webrepl_setup`, modify boot.py on your Lolin S2 mini as suggested at https://www.techcoil.com/blog/how-to-setup-micropython-webrepl-on-your-esp32-development-board/. This should enable you to access the S2 mini via your own network rather than needing to connect to an access point on the S2 mini.
